Details popup window

hakichris opened the pull request:
Detail Popup 

in this project we are creating a card details of the project card to popup when it is clicked on .

hakichris committed:
Update css

hakichris committed:
Update html

hakichris committed:
Update html

hakichris committed:
Update css

hakichris committed:
Update css

hakichris committed:
Update css

Commit:
Fix modal bugs

hakichris committed:
Update desktop

hakichris committed:
Update index.html

Commit:
Resolved blockers from script file

Commit:
Merge branch 'detailsPopupWindow' of https://github.com/hakichris/Christian_Portfolio.github.io into detailsPopupWindow

hakichris committed:
Update css

hakichris committed:
Update js

hakichris committed:
Update js

hakichris committed:
Update html

hakichris committed:
Update html

hakichris committed:
Update css

hakichris committed:
Update js

hakichris committed:
Update css

hakichris committed:
Update js

hakichris committed:
Update images

hakichris committed:
Update js

Commit:
Update Js file

Commit:
Merge branch 'detailsPopupWindow' of https://github.com/hakichris/Christian_Portfolio.github.io into detailsPopupWindow

Commit:
Updated js file with close modal functionality

hakichris committed:
Update css

hakichris committed:
Update html

hakichris committed:
Update css

hakichris committed:
Created Validation js

hakichris committed:
Update css

hakichris committed:
Update Js

hakichris committed:
Update css

hakichris committed:
Update js

hakichris committed:
Update css

hakichris committed:
Fix linters

hakichris committed:
Fix linters

hakichris committed:
Fix linters

hakichris committed:
Fix linter

hakichris committed:
Fix linters

hakichris committed:
Fix linters

hakichris committed:
Fix linters

hakichris committed:
Fix linters

hakichris committed:
Updata js

hakichris committed:
Update js

hakichris committed:
Update js

hakichris committed:
Fix linters

hakichris committed:
Update js

hakichris committed:
Fix linters

hakichris committed:
fix linters

hakichris committed:
Update js

AnselemOdims commented on index.html:
```diff
@@ -79,271 +80,247 @@ <h2 id="connect">LET'S CONNECT</h2>
             <ul class="client">
-              <li id="h3">CANOPY</li>
+              <li id="h3"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt="" />
               </li>
 
-              <li class="langue">Back end Dev</li>
+              <li class="langue"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt=""/>
               </li>
 
-              <li class="year">2015</li>
+              <li class="year"> </li>
             </ul>
             <p class="project_description">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
+           
             </p>
             <ul class="code_language">
-              <li class="html"><a class="ht" href="#">html</a></li>
-              <li class="css"><a class="ht" href="#">css</a></li>
-              <li class="javascript"><a class="ht" href="#">javascript</a></li>
+              <li ><a class="ht html" href="#"> </a></li>
+              <li ><a class="ht css" href="#"> </a></li>
+              <li ><a class="ht javascript" href="#"> </a></li>
```
- [x] Please kindly ensure that the markup for the works/projects section is completely removed so that it can be generated dynamically with javascript from the projects array created.
- [x] Also, at the moment the modal is not created dynamically and it is only one modal for all projects. Please kindly ensure it is dynamically generated with Javascript when a project button is clicked.

hakichris replied:
Resolved, thank you.

hakichris replied:
Resolved. thank you.

AnselemOdims commented on Desktop.css:
```diff
@@ -182,7 +183,7 @@ main {
 }
 
-.work {
+.works {
```
- [x] Good job styling the modal, but your design does not match the original design guidelines. Please kindly ensure that your design matches the original design guidelines(color, typography, spacing, layout etc)
- [x] Also, the modal is not responsive on the mobile because it has a fixed with. Please kindly ensure it is responsive on both desktop and mobile(different screens)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/210798710-b37fa62d-9993-41ce-a47b-488f5c4165f7.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/210798869-d9127bca-2b1f-4347-926b-23d9426e69dd.png)

hakichris replied:
Thank you so much for you feedback. I have corrected the design. it is now responsive and match the figma design as close as possible.

hakichris replied:
In case, you don't see my partner as participant, it will not be my fault. his commits are still there. buy somehow github is not showing his profile. Thank you for understanding

OyePriscilla replied:
- [x] Awesome working on some of the required changes by the previous reviewer, please I noticed that the pop-up for the desktop version is not displayed correctly according to the design. The two button elements are outside the parent container. Kindly fix it. 👍 
- [x] Please ensure that the correct content is popped-up when `See project` event is triggered.
- [x] You may need to align the design according to the template. 👍 

## Your Design
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text-removebg-preview (1)](https://user-images.githubusercontent.com/103386183/211027396-ae0e6cc4-7430-40cd-949a-aeb40f97a0be.png)

AnselemOdims reviewed (changes requested):
### Hi @hakichris , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

hakichris committed:
Update html

hakichris committed:
Update css

hakichris committed:
Update css

hakichris committed:
Update css

Commit:
Fixed modal display bugs

hakichris committed:
Update html

hakichris committed:
Update css

hakichris committed:
Update js

hakichris committed:
Update html

Commit:
Update css file

Commit:
update js

Commit:
Update Js file

hakichris committed:
Update  css

hakichris committed:
Update js

hakichris committed:
Update picture file

hakichris committed:
Update css

hakichris committed:
Update html

hakichris committed:
Update Validation

hakichris committed:
Update mobile

hakichris committed:
Update popup

hakichris committed:
Update html

hakichris committed:
Update css

hakichris committed:
Update css

OyePriscilla reviewed (changes requested):
Hi @hakichris 😃, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425615-8927e377-c543-407b-8da4-5c09fecc261b.gif)

## Highlights
Github flow observed✔️
App worked as expected ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hakichris committed:
Update css

hakichris committed:
Update css

hakichris commented:
Hello @OyePriscilla  thank you for your kind feedback. I fixed the Button being outside the scope. 
the content was not part of the requirement however, i did my best to meet the figma design even though i changed the content on my design like the picture is stretched. i would appreciate if you don't consider that to block the project. the correct content popup when you click see project according to my content. thank you let me know if i can submit another review.

hakichris committed:
Update html

hakichris commented:
The linters are working too.

leonardodiasb commented on index.html:
```diff
@@ -79,271 +80,240 @@ <h2 id="connect">LET'S CONNECT</h2>
-                <li id="h3">Uber</li>
+                <li id="h3"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt="" />
               </li>
 
-              <li class="langue">Lead Developer</li>
+              <li class="langue"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt="" />
               </li>
 
-              <li class="year">2018</li>
+              <li class="year"> </li>
             </ul>
             <p class="project_description">
               A daily selection of privately personalized reads; no accounts or
               sign-ups required.
             </p>
             <ul class="code_language">
-              <li class="ht1"><a class="html" href="#">html</a></li>
+              <li class="ht1"><a class="html" href="#"> </a></li>
               <li class="ht2"><a class="ruby" href="#">Ruby on rails</a></li>
-              <li class="ht3"><a class="css" href="#">css</a></li>
-              <li class="ht4"><a class="javascript" href="#">javascript</a></li>
+              <li class="ht3"><a class="css" href="#"> </a></li>
+              <li class="ht4"><a class="javascript" href="#"> </a></li>
             </ul>
-            <div class="link_to_project">
-              <a class="check_Project" href="#">See Project</a>
-            </div>
+            <button class="link_to_project" id="4"> See Project </button>
           </div>
         </div>
-      </div>
-     
-
-
-
+      </div>f 
```
- [OPTIONAL] Since the previous code reviewer forgot to mention this, I have to put it as an 'OPTIONAL' requirement. But all of the HTML code related to the projects section should be removed and refactored so you create this section dynamically with JavaScript. I see that part of the content is created dynamically, but the hard-coded HTML should be removed. Kindly be aware of this and fix this for the next submission.
 - 
<img width="999" alt="image" src="https://user-images.githubusercontent.com/61919617/211044895-b6a277df-99fc-4b9c-ae11-a2cba1555418.png">

 - [ ] Your modal is almost perfect for the project, but unfortunately, it does not pass on the requirements. The modal should not be transparent, and on the desktop version, it should be displayed in the center of the screen. Kindly fix these issues to make a better user experience and looks more similar to the Figma template.
 - Your design:
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/61919617/211045994-573bc213-6842-43fa-937d-4066b57f2d93.png">

leonardodiasb reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi @hakichris, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 

 - Github flow observed✔️
 - App worked as expected ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hakichris committed:
Update mobile

hakichris committed:
Update html

hakichris committed:
Update mobi

hakichris committed:
Update js

hakichris committed:
Update js

nicupop729 reviewed (commented):
Hello @hakichris and @JansueT 👋,

_This is an additional review as a result of the second opinion request._

I appreciate a lot your hard work on this pair-programming activity! :clap:

Kindly note that if you don't agree with a reviewer, is expected from you to ask for a Second Opinion Review before pushing other commits. Also, after you asked for a Second Opinion Review, you need to wait to receive it before working again on the project. Unfortunately, it was not the case in this pull request.

The comment that the last reviewer made about rendering the work cards dynamically with JavaScript, even though it was a very important project requirement, please take it, as he mentioned, as Optional, because the second reviewer, @OyePriscilla, missed mentioning it again.

But the second aspect mentioned by the last reviewer, regarding the look of your modal, kindly take it as mandatory, as it was mentioned by all reviewers.
![image](https://user-images.githubusercontent.com/83596135/211085114-2be189e1-b078-43e2-b8ff-e23a42b44f35.png)

The project was not approved because this aspect was asked as mandatory and not as optional.

Please take the necessary refactoring in order for the modal to look as it looks in the Figma design, and when you feel ready, kindly ask again for another review from your dashboard.

Keep rocking 🚀!

nicupop729 commented:
At this moment, the PR is to be invalidated, because linter checks are not running.
Kindly, refactor your modal to look as close as possible to the Figma design and also, please resolve the conflicts in order for the linters to run.
Otherwise, the next reviewer will have to invalidate your Code Review Request.

Commit:
Resolved issues from feedback received

Commit:
Updated the modal design

hakichris committed:
Update js

hakichris committed:
Update js

hakichris committed:
Update js
